### PR TITLE
Add `preload` option

### LINKED ISSUE #48: Preload script before `goto`
```js
if(options.preload){
  await page.evaluateOnNewDocument(options.preload);
}

await page.goto(...);
```

### PR DESCRIPTION
**Pull request by brandon93s**
Fixes #48

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -136,6 +136,7 @@ const captureWebsite = async (input, options) => {
 		launchOptions: {},
 		_keepAlive: false,
 		isJavaScriptEnabled: true,
+		preload: undefined,
```
Comment (sindresorhus) [inline index.js:139]: This is moot.

File: readme.md [hunk 2]
```diff
@@ -442,6 +442,13 @@ Overwrite the destination file if it exists instead of throwing an error.
 
 *This option applies only to `captureWebsite.file()`.*
 
+##### preload
```
Comment (sindresorhus) [inline readme.md:445]: `preload` sounds like it could be a boolean that sets whether to preload it.

File: readme.md [hunk 3]
```diff
@@ -442,6 +442,13 @@ Overwrite the destination file if it exists instead of throwing an error.
...
+
+Type: `string | Function`\
+Default: `undefined`
+
+Inject a function to be executed prior to navigation; useful for [altering the JavaScript environment](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageevaluateonnewdocumentpagefunction-args).
```
Comment (sindresorhus) [inline readme.md:450]: Can you mention a few common use-cases? Just for lazy people.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Bump :)

**Comment by sindresorhus** [comment]
Would you be willing to add this to the CLI version too?

**Comment by brandon93s** [comment]
Moved to https://github.com/sindresorhus/capture-website/pull/76.

**Pull request closed without merging**